app/code/Magento/Checkout/Test/Mftf/Test/StorefrontCheckRenderingClearCartButtonOnTheCartPageBasedOnStoresConfigurationTest.xml

```diff
@@ -0,0 +1,73 @@
+            <severity value="MAJOR"/>
+            <group value="shoppingCart"/>
+        </annotations>
+
+        <before>
+            <!-- Create simple product -->
+            <createData entity="SimpleProduct2" stepKey="createProduct"/>
+        </before>
+        <after>
+            <!-- Delete simple product -->
+            <deleteData createDataKey="createProduct" stepKey="deleteProduct"/>
+
+            <!-- Disable rendering of clear shopping cart button on the cart page -->
+            <magentoCLI command="config:set {{DisableClearShoppingCartButtonOnTheCartPage.path}} {{DisableClearShoppingCartButtonOnTheCartPage.value}}" stepKey="disableClearShoppingCart"/>
+
+            <!-- Log out -->
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+        <!-- Add product to cart -->
+        <actionGroup ref="OpenStoreFrontProductPageActionGroup" stepKey="openProductPage">
+            <argument name="productUrlKey" value="$$createProduct.custom_attributes[url_key]$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddProductToCartActionGroup" stepKey="addProductToCart">
+            <argument name="product" value="$$createProduct$$"/>
+            <argument name="productCount" value="1"/>
+        </actionGroup>
+
+        <!-- Navigate to cart page -->
+        <actionGroup ref="StorefrontOpenCartFromMinicartActionGroup" stepKey="openShoppingCart"/>
+        <waitForPageLoad stepKey="waitForShoppingCartLoad" />
+
+        <!-- Assert that empty cart button is not rendered on the cart page -->
+        <dontSeeElement selector="{{CheckoutCartProductSection.emptyCartButton}}" stepKey="dontSeeClearShoppingCartButton"/>
+
+        <!-- Open new browser's window and login as Admin -->
+        <openNewTab stepKey="openNewTab"/>
+        <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+
+        <!-- Navigate to checkout cart configuration --> 
+        <actionGroup ref="AdminOpenSalesCheckoutConfigPageActionGroup" stepKey="openCheckoutCartConfig">
+            <argument name="tabGroupAnchor" value="#checkout_cart-link"/>
+        </actionGroup>
+
+        <!-- Enable clear shopping cart button -->
+        <actionGroup ref="AdminCheckoutClearShoppingCartEnabledActionGroup" stepKey="enableClearShoppingCartButton"/>
+
+        <!-- Flush cache -->
+        <magentoCLI command="cache:flush" stepKey="cacheFlush"/>
+
+        <!-- Back to the Cart page and refresh the page -->
+        <switchToPreviousTab stepKey="switchToPreviousTab"/>
+        <reloadPage stepKey="refreshPage"/>
+        <waitForPageLoad stepKey="waitPageReload"/>
+
+        <!-- Assert that empty cart button is rendered on the cart page -->
+        <seeElement selector="{{CheckoutCartProductSection.emptyCartButton}}" stepKey="SeeClearShoppingCartButton"/>
```

Comment:
This test only covers the emptyCartButton appears when configuration is enabled. Additonal steps needs to be added to cover modal.